### DISCUSSION TITLE
Learning TE

### DISCUSSION THREAD

**Discussion by RobMoen** [mdbootstrap/TW-Elements#1959] [Support from community]
Good day,
I am new to Tailwind Elements but managed to install everything using the great tutorial from Michael. I used the vite method. I created 2 html files, index.html and about.html and within the dev server it all works fine. However, when I use the command "npm run build" it generates only the index.html and not about.html. My tailwind.config looks like this:

/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "./index.html",
				"./about.html",
    './components/**/*.{html,js}',
    "./src/**/*.{js,ts,jsx,tsx}",
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}

What am I doing wrong? Thanks for helping!
Best regards, Rob

**Comment by juujisai** [comment]
Hi! Checkout the vite documentation about Building an app for Production

https://vitejs.dev/guide/build.html
